host/xtest/ftpm_ta.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024 Linaro Limited
+ *
+ * The copyright in this software is being made available under the BSD License,
+ * included below. This software may be subject to other third party and
```

Comment:
"included below" should be rephrased :(
Maybe:
```
 * The copyright in this software is being made available under the BSD-2-Clause
 * License. This software may be subject to other third party and
 * ...

Reply:
Argh, that's changing the license text. I'll restore the removed portion.